Doc/library/fnmatch.rst

```diff
@@ -75,7 +75,7 @@ patterns.
 
 .. function:: filter(names, pattern)
 
-   Return the subset of the list of *names* that match *pattern*. It is the same as
+   Return a list of items in iterable *names* that match *pattern*. It is the same as
```

Comment:
I would be more inclined to say `Return a list of items from *names* that match *pattern*.`  I don't think the usage of the word `list` in the original necessary meant a Python list type, just that names was a sequence of one or more items.  The next line with the equivalency is enough to show that `names` can be an iterable.

Reply:
Hmm, I'd say that whenever it is simple enough, it's better to document what's is expected and whas is returned in the API description, instead of examples. IMHO, examples should just illustrate not give actual information.

Reply:
@csabella Any thought regarding my last comment?

Reply:
@iritkatriel I like it. Just updated the PR with that change.